src/frontend/Felles/PersonHeader/PersonHeader.tsx

```diff
@@ -164,7 +165,7 @@ const PersonHeaderComponent: FC<{ data: IPersonopplysninger; behandling?: Behand
                             gåTilUrl(`/person/${fagsakPersonId}`);
                         }}
                     >
-                        <Visningsnavn>{visningsnavn}</Visningsnavn>
+                        <Visningsnavn size={'small'}>{visningsnavn}</Visningsnavn>
```

Comment:
Burde vi bruke BodyShortSmall i stedet for å bruke size=small? 👀

Reply:
> ![image](https://user-images.githubusercontent.com/937168/205233950-749d7f27-e7b5-4700-9ede-ce42b200e501.png)

Til nå - by far kjedeligste oppgaven jeg har løst 😂 

Gruer meg til v.2 når alle `navds-...` må byttes ut med `a-...`. Da er det mulig jeg tar meg en uke ferie 😎